common/configuration-db.adoc

```diff
@@ -12,23 +12,32 @@ Before installing an internal database, add a disk to the infrastructure hosting
 * If this is the first {product-title} appliance, choose *1) Create key*.
 * If this is not the first {product-title} appliance, choose *2) Fetch key* from remote machine to fetch the key from the first {product-title} appliance. All {product-title} appliances in a multi-region deployment must use the same key.
 . Choose *1) Internal* for the database location.
-. Choose a disk for the database. For example:
+. Choose a disk for the database. This can be either a disk you attached previously, or a partition on the current disk. 
++
+[IMPORTANT]
+====
+Red Hat recommends using a separate disk for the database.
+====
++
+If there is an unpartitioned disk attached to the virtual machine, the dialog will show options similar to the following:
 +
 ----
-1)  /dev/vdb: 20480
-
-Choose disk:
+1) /dev/vdb: 20480
+2) Don't partition the disk 
 ----
 +
-Enter *1* to choose `/dev/vdb` for the database location.
-
-. When prompted, enter a unique three digit region ID to create a new region.
+* Enter *1* to choose `/dev/vdb` for the database location. This option creates a logical volume using this device and mounts the volume to the appliance in a location appropriate for storing the database. The default location is `/var/opt/rh/rh-postgresql95/lib/pgsql`, which can be found in the environment variable `$APPLIANCE_PG_MOUNT_POINT`.
+* Enter *2* to continue without partitioning the disk. A second prompt will confirm this choice. Selecting this option results in using the root filesystem for the data directory (not advised in most cases). 
+. Enter *Y* or *N* for *Configure this server as a dedicated database instance?*
+* Select *Y* to configure the appliance _only_ as a database. The resulting appliance will not have a user interface. This option is the default, and also can be chosen by pressing *Enter*.
+* Select *N* to configure the appliance with a user interface.
```

Comment:
@carbonin I see what you're saying. We refer to the graphical CloudForms accessed through a web browser as "the user interface" (lower case) in the docs, so I would think it would be understandable to readers. However, I was unaware of what was actually happening in the background. I learned that in the SSUI guide, we occasionally differentiate UI/SSUI with "the full administrative user interface". What do you think of:

* Select *Y* to configure the appliance _only_ as a database. As a result, the appliance is configured as a basic PostgreSQL server, without a user interface. 
* Select *N* to configure the appliance with the full administrative user interface.

When you say "the application", are you referring to the appliance?